packages/engine/Source/Shaders/ShadowVolumeAppearanceFS.glsl

```diff
@@ -65,21 +65,6 @@ void main(void)
 #endif // SPHERICAL
 #endif // TEXTURE_COORDINATES
 
-#ifdef PICK
-#ifdef CULL_FRAGMENTS
-    // When classifying translucent geometry, logDepthOrDepth == 0.0
-    // indicates a region that should not be classified, possibly due to there
-    // being opaque pixels there in another buffer.
-    // Check for logDepthOrDepth != 0.0 to make sure this should be classified.
-    if (0.0 <= uv.x && uv.x <= 1.0 && 0.0 <= uv.y && uv.y <= 1.0 || logDepthOrDepth != 0.0) {
```

Comment:
This condition should have been instead:

```js
if (0.0 <= uv.x && uv.x <= 1.0 && 0.0 <= uv.y && uv.y <= 1.0 && logDepthOrDepth != 0.0) {
```

Which is equivalent to:
```js
if (!(uv.x <= 0.0 || 1.0 <= uv.x || uv.y <= 0.0 || 1.0 <= uv.y || logDepthOrDepth == 0.0)) {
```

But its safer to extract the CULL_FRAGMENTS to occur prior to either the pick/regular path, so that there is a unified discard for both paths, and avoid making these inverse logic problems.

Then the pick path can safely explicitly set `out_FragColor.a = 1.0`, and call `czm_writeDepthClamp()` just based on the `CULL_FRAGMENTS` path.